<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QMainWindow" name="Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>960</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>960</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>960</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>月時計 ~Lunar Dial~</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resources/resources.qrc">
    <normaloff>:/res/img/Logo1x1.png</normaloff>:/res/img/Logo1x1.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>0.995000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>60</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>i</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>541</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="TopBar">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>12</number>
     </property>
     <property name="rightMargin">
      <number>12</number>
     </property>
     <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
      <widget class="QLabel" name="Logo">
       <property name="minimumSize">
        <size>
         <width>72</width>
         <height>36</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>72</width>
         <height>36</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>GLT-GonunneObsolete</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>月時計</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="closeBtn" name="closeBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="tabletTracking">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>×</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>371</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="Time">
     <item>
      <widget class="QLCDNumber" name="nowTimeH">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="smallDecimalPoint">
        <bool>false</bool>
       </property>
       <property name="digitCount">
        <number>2</number>
       </property>
       <property name="mode">
        <enum>QLCDNumber::Dec</enum>
       </property>
       <property name="value" stdset="0">
        <double>12.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="timeSep1">
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>48</pointsize>
        </font>
       </property>
       <property name="text">
        <string>:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="nowTimeM">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="smallDecimalPoint">
        <bool>false</bool>
       </property>
       <property name="digitCount">
        <number>2</number>
       </property>
       <property name="value" stdset="0">
        <double>12.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="timeSep2">
       <property name="minimumSize">
        <size>
         <width>2</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>48</pointsize>
        </font>
       </property>
       <property name="text">
        <string>:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="nowTimeS">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="smallDecimalPoint">
        <bool>false</bool>
       </property>
       <property name="digitCount">
        <number>2</number>
       </property>
       <property name="mode">
        <enum>QLCDNumber::Dec</enum>
       </property>
       <property name="value" stdset="0">
        <double>12.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>501</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="mainCalendar">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="1" column="0">
      <widget class="dayLabel" name="c11">
       <property name="text">
        <string>00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="dayLabel" name="c33">
       <property name="text">
        <string>14</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="dayLabel" name="c44">
       <property name="text">
        <string>22</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="dayLabel" name="c13">
       <property name="text">
        <string>00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="dayLabel" name="c23">
       <property name="text">
        <string>7</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="dayLabel" name="c24">
       <property name="text">
        <string>8</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="c06">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>64</height>
        </size>
       </property>
       <property name="text">
        <string>周六
土曜日</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="dayLabel" name="c32">
       <property name="text">
        <string>13</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="dayLabel" name="c35">
       <property name="text">
        <string>16</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="dayLabel" name="c22">
       <property name="text">
        <string>6</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="dayLabel" name="c42">
       <property name="text">
        <string>20</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="dayLabel" name="c31">
       <property name="text">
        <string>12</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="dayLabel" name="c52">
       <property name="text">
        <string>27</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="dayLabel" name="c16">
       <property name="text">
        <string>3</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="dayLabel" name="c26">
       <property name="text">
        <string>10</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="dayLabel" name="c34">
       <property name="text">
        <string>15</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="dayLabel" name="c21">
       <property name="text">
        <string>5</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="dayLabel" name="c53">
       <property name="text">
        <string>28</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="dayLabel" name="c43">
       <property name="text">
        <string>21</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="dayLabel" name="c27">
       <property name="text">
        <string>11</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="6">
      <widget class="dayLabel" name="c57">
       <property name="text">
        <string>00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="dayLabel" name="c51">
       <property name="text">
        <string>26</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="dayLabel" name="c54">
       <property name="text">
        <string>29</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="6">
      <widget class="dayLabel" name="c47">
       <property name="text">
        <string>25</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="c02">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>64</height>
        </size>
       </property>
       <property name="text">
        <string>周二
火曜日</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="dayLabel" name="c37">
       <property name="text">
        <string>18</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="dayLabel" name="c55">
       <property name="text">
        <string>30</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="dayLabel" name="c45">
       <property name="text">
        <string>23</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="c01">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>64</height>
        </size>
       </property>
       <property name="text">
        <string>周一
月曜日</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="c03">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>64</height>
        </size>
       </property>
       <property name="text">
        <string>周三
水曜日</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="dayLabel" name="c12">
       <property name="text">
        <string>00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="dayLabel" name="c25">
       <property name="text">
        <string>9</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="dayLabel" name="c56">
       <property name="text">
        <string>00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="dayLabel" name="c14">
       <property name="text">
        <string>1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="c07">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>64</height>
        </size>
       </property>
       <property name="text">
        <string>周日
日曜日</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="dayLabel" name="c36">
       <property name="text">
        <string>17</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="c04">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>64</height>
        </size>
       </property>
       <property name="text">
        <string>周四
木曜日</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="dayLabel" name="c41">
       <property name="text">
        <string>19</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="dayLabel" name="c46">
       <property name="text">
        <string>24</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="dayLabel" name="c15">
       <property name="text">
        <string>2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="dayLabel" name="c17">
       <property name="text">
        <string>4</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="c05">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>64</height>
        </size>
       </property>
       <property name="text">
        <string>周五
金曜日</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="lastYear">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lastMonth">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextMonth">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>170</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextYear">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>170</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QListWidget" name="Reminders">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>690</y>
      <width>321</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addReminder">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>920</y>
      <width>321</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="weatherView" name="weather">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>690</y>
      <width>181</width>
      <height>251</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>181</width>
      <height>251</height>
     </size>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="url">
     <url>
      <string>qrc:/res/web/weather.html</string>
     </url>
    </property>
    <property name="zoomFactor">
     <double>0.850000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="todayDate">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>110</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backToday">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>回到今日</string>
    </property>
   </widget>
   <widget class="QLabel" name="calYear">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="calMonth">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>170</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="bgImage" name="bgImg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>541</width>
      <height>921</height>
     </rect>
    </property>
    <property name="pixmap">
     <pixmap resource="../Resources/resources.qrc">:/res/img/bg.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>bgImg</zorder>
   <zorder>pushButton</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>horizontalLayoutWidget_2</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>lastYear</zorder>
   <zorder>lastMonth</zorder>
   <zorder>nextMonth</zorder>
   <zorder>nextYear</zorder>
   <zorder>Reminders</zorder>
   <zorder>addReminder</zorder>
   <zorder>weather</zorder>
   <zorder>todayDate</zorder>
   <zorder>backToday</zorder>
   <zorder>calYear</zorder>
   <zorder>calMonth</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
  <customwidget>
   <class>closeBtn</class>
   <extends>QLabel</extends>
   <header location="global">View.ExtendedClasses</header>
  </customwidget>
  <customwidget>
   <class>dayLabel</class>
   <extends>QLabel</extends>
   <header location="global">View.ExtendedClasses</header>
  </customwidget>
  <customwidget>
   <class>weatherView</class>
   <extends>QWebEngineView</extends>
   <header location="global">View.ExtendedClasses</header>
  </customwidget>
  <customwidget>
   <class>bgImage</class>
   <extends>QLabel</extends>
   <header location="global">View.ExtendedClasses</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Home</receiver>
   <slot>toAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>410</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addReminder</sender>
   <signal>clicked()</signal>
   <receiver>Home</receiver>
   <slot>toAddReminder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>927</y>
    </hint>
    <hint type="destinationlabel">
     <x>10</x>
     <y>870</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backToday</sender>
   <signal>clicked()</signal>
   <receiver>Home</receiver>
   <slot>backToToday()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextMonth</sender>
   <signal>clicked()</signal>
   <receiver>Home</receiver>
   <slot>nextMonth()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>444</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextYear</sender>
   <signal>clicked()</signal>
   <receiver>Home</receiver>
   <slot>nextYear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>497</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>533</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lastMonth</sender>
   <signal>clicked()</signal>
   <receiver>Home</receiver>
   <slot>lastMonth()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lastYear</sender>
   <signal>clicked()</signal>
   <receiver>Home</receiver>
   <slot>lastYear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>17</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>toAbout()</slot>
  <slot>toAddReminder()</slot>
  <slot>backToToday()</slot>
  <slot>nextMonth()</slot>
  <slot>nextYear()</slot>
  <slot>lastMonth()</slot>
  <slot>lastYear()</slot>
 </slots>
</ui>
